.github/workflows/ci.yaml

```diff
@@ -134,3 +134,35 @@ jobs:
           token: ${{ secrets.CODECOV_TOKEN }}
           flags: unittests
           file: cover.out
+  # See: https://google.github.io/oss-fuzz/getting-started/continuous-integration/
+  Fuzzing:
+    runs-on: ubuntu-latest
+    permissions:
+      security-events: write
+    steps:
+      - name: Build Fuzzers
+        id: build
+        uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
```

Comment:
plz do not use master version dependency, use dependencies with specific versions, plz reference other dependencies in this file, e.g. 
codecov/codecov-action@0565863a31f2c772f9f0395002a31e3f06189574 # v5.4.0

Reply:
> plz do not use master version dependency, use dependencies with specific versions, plz reference other dependencies in this file, e.g. [codecov/codecov-action@0565863](https://github.com/codecov/codecov-action/commit/0565863a31f2c772f9f0395002a31e3f06189574) # v5.4.0

done